pubspec.yaml

```diff
@@ -1,18 +1,17 @@
 name: shelf_web_socket
-version: 0.2.2+4
+version: 0.2.2+5
 
 description: A WebSocket handler for Shelf.
```

Comment:
Could you also make this a bit longer? See https://pub.dartlang.org/packages/shelf_web_socket#-analysis-tab-

Reply:
I guess? Seem arbitrary.

Reply:
Optimizing for Google SEO